packages/netlify-cms-backend-azure/src/API.ts

```diff
@@ -0,0 +1,782 @@
+      commit.changes.delete(path);
+
+      const push = new AzurePush(ref);
+      push.commits.push(commit);
+
+      return this.requestJSON({
+        url: `${this.endpointUrl}/pushes`,
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json; charset=utf-8' },
+        body: JSON.stringify(push),
+      });
+    });
+  }
+
+  generateContentKey(collectionName: string, slug: string) {
+    return generateContentKey(collectionName, slug);
+  }
+
+  contentKeyFromBranch(branch: string) {
+    return branch.substring(`${CMS_BRANCH_PREFIX}/`.length);
+  }
+
+  branchFromContentKey(contentKey: string) {
+    return `${CMS_BRANCH_PREFIX}/${contentKey}`;
+  }
+
+  async getMergeRequests(sourceBranch?: string) {
+    const mergeRequests = await this.requestJSON({
+      url: `${this.endpointUrl}/pullrequests`,
+      params: {
+        'searchCriteria.status': 'active',
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        'searchCriteria.targetRefName': this.branchToRef(this.branch),
+        'searchCriteria.includeLinks': false,
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        ...(sourceBranch ? { 'searchCriteria.sourceRefName': this.branchToRef(sourceBranch) } : {}),
+      },
+    });
+
+    return mergeRequests.value.filter((mr: any) =>
+      mr.sourceRefName.startsWith(this.branchToRef(CMS_BRANCH_PREFIX)),
+    );
+  }
+
+  /**
+   * Gets a list of all unpublished branches, which is a list of the pending
+   * merge requests projected to just their source branch names.
+   */
+  async listUnpublishedBranches(): Promise<string[]> {
+    const mergeRequests = await this.getMergeRequests();
+    const branches = mergeRequests.map((mr: any) => this.refToBranch(mr.sourceRefName));
+    return branches;
+  }
+
+  async isFileExists(path: string, branch: string) {
+    return await this.requestText({
+      url: `${this.endpointUrl}/items/`,
+      params: { version: branch, path },
+      cache: 'no-store',
+    })
+      .then(() => {
+        return true;
+      })
+      .catch(error => {
+        if (error instanceof APIError && error.status === 404) {
+          return false;
+        }
+        throw error;
+      });
+  }
+
+  /**
+   * Creates a new pull request with a label of "draft", based on the target branch.
+   * See documentation at: https://docs.microsoft.com/en-us/rest/api/azure/devops/git/pull%20requests/create?view=azure-devops-rest-5.1
+   * @param branch The branch to create the pull request from.
+   * @param commitMessage A message to use as the title for the pull request.
+   * @param status The status of the pull request.
+   */
+  async createPullRequest(branch: string, commitMessage: string, status: string) {
+    const pr = {
+      sourceRefName: this.branchToRef(branch),
+      targetRefName: this.branchToRef(this.branch),
+      title: commitMessage,
+      description: `Editorial workflow to manage approval and merge of ${commitMessage}`,
```

Comment:
There is a `DEFAULT_PR_BODY` constant that we're using as description:
https://github.com/netlify/netlify-cms/blob/684b79e43bebb63ce1e844eae5c8c0e76087687b/packages/netlify-cms-lib-util/src/API.ts#L2

Reply:
Updated. More widely, should this get bounced through the translation API?

Reply:
You're right, we should support an option to translate it. As this issue is relevant for other backends too I think it is out of scope for this PR.